kinds/dashboard/dashboard_kind.cue

```diff
@@ -574,10 +574,10 @@ lineage: schemas: [{
 			libraryPanel?: #LibraryPanelRef
 
 			// It depends on the panel plugin. They are specified by the Options field in panel plugin schemas.
-			options: {...} @grafanamaturity(NeedsExpertReview)
+			options?: {...} @grafanamaturity(NeedsExpertReview)
```

Comment:
🤔 I think we can remove the `@grafanamaturity(NeedsExpertReview)`, as we are confirming that this is the expected definition of the prop, right?